### DISCUSSION TITLE
Some ideas

### DISCUSSION THREAD

**Discussion by gand07** [Daveiano/weewx-wdc#223] [Ideas]
Option to move the left menu to the top, it will give more space for tiles, tiles and tables when the page is displayed on laptop/desktop. 

Ability to configure the size of the charts so that there can be several next to each other on laptop/desktop. 

Thank you for a clear template and the good support

Gert

**Comment by Daveiano** [comment]
> Option to move the left menu to the top, it will give more space for tiles, tiles and tables when the page is displayed on laptop/desktop.

The classic layout uses a "slide in/slide out" mode for the left menu. Perhaps this could also be an option? See https://www.weewx-hbt.de/classic/ for an example.

> Ability to configure the size of the charts so that there can be several next to each other on laptop/desktop.

Currently, on desktop (larger screens) there are 3 charts shown in a row. You want to configure something like "show 4 charts in a row", right?

> Thank you for a clear template and the good support

Thank you! For your great ideas and suggestions!
